mirdata/datasets/compmusic_indian_tonic.py

```diff
@@ -0,0 +1,262 @@
+def load_audio(audio_path):
+    """Load a Indian Art Music Tonic audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+    """
+    return librosa.load(audio_path, sr=44100, mono=False)
+
+
+@core.docstring_inherit(core.Dataset)
+class Dataset(core.Dataset):
+    """
+    The compmusic_indian_tonic dataset
+    """
+
+    def __init__(self, data_home=None, version="default"):
+        super().__init__(
+            data_home,
+            version,
+            name="compmusic_indian_tonic",
+            track_class=Track,
+            bibtex=BIBTEX,
+            indexes=INDEXES,
+            remotes=REMOTES,
+            download_info=DOWNLOAD_INFO,
+            license_info=LICENSE_INFO,
+        )
+
+    @core.cached_property
+    def _metadata(self):
+        centers = ["CM", "IISc", "IITM"]
+        meta_files = []
+        for center in centers:
+            meta_files = meta_files + glob.glob(
+                os.path.join(
+                    self.data_home,
+                    "indian_art_music_tonic_1.0",
+                    center,
+                    "annotations",
+                    "*.json",
+                )
+            )
+        meta_files = [x for x in meta_files if "IITM1" not in x]
+
+        metadata = {}
+        for meta in meta_files:
+            try:
+                with open(meta, "r") as fhandle:
+                    data = json.load(fhandle)
+                    if "IITM" not in meta:
+                        for k in list(data.keys()):
+                            idx = k.split("/")[-1].replace(".mp3", "")
+                            metadata[idx] = {
+                                "tonic": float(data[k]["tonic"]),
+                                "artist": data[k]["artist"],
+                                "gender": data[k]["gender"],
+                                "mbid": data[k]["mbid"],
+                                "type": data[k]["type"],
+                                "tradition": data[k]["tradition"],
+                            }
+                    else:
+                        for k in list(data.keys()):
+                            idx = k.split("/")[-1].replace(".mp3", "")
+                            metadata[idx] = {
+                                "tonic": float(data[k]["tonic"]),
+                                "artist": data[k]["artist"],
+                                "gender": data[k]["gender"],
+                                "mbid": data[k]["mbid"],
+                                "type": data[k]["type"],
+                                "tradition": data[k]["tradition"],
+                            }
+
+            except FileNotFoundError:
+                raise FileNotFoundError("Metadata not found. Did you run .download()?")
```

Comment:
this is not covered again. maybe add a track with missing metadata and check if you expect this error

Reply:
Ok! Will do that ASAP :)